package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "alexwayfer.name",
+	"name": "d20201217-4-n2ajin",
```

Comment:
Hey @AlexWayfer, Jan from Depfu here. I've just updated this pull request with our experimental npm7 support but as you can see it has room for improvement. The reason this weird random name appears is because you removed the name from package.json and npm then uses the directory name of where the package.json lies (which is a tmpdir in our case). We're going to fix that, even though I think it is not particularly relevant but it will cause diffs here every time and that's not useful.

Reply:
@AlexWayfer This seems to be resolved now. Please let us know if you see issues with the PR in it's current state.

Reply:
@halfbyte thank you! Especially for information and updates right here. Everything looks good to me.